scripts/parse_registries.py

```diff
@@ -19,20 +19,14 @@
  * github organization.
  ***************************************************************/"""
 
-REGISTRY_HEADER = (
```

Comment:
Is this a upstream commit ?

Reply:
Let's undo this commit since https://github.com/ibm-openbmc/openbmc-build-scripts/pull/7 merged